tools/mo/openvino/tools/mo/ops/split.py

```diff
@@ -98,7 +98,7 @@ def infer(node):
         # value propagation
         input_value = node.in_port(0).data.get_value()
         if input_value is not None:
-            split = np.split(input_value, idxs[:-1], axis)
+            split = np.split(input_value, mo_array(idxs[:-1], dtype=np.int), axis)
```

Comment:
Could you please use the same type as `split_lengths`? Not sure that `np.int` can be sufficient for idx in case huge embedding table

Reply:
dtype of this argument does not influence output dtype of `split` variable and specifying np.int is same to specify np.int64
![image](https://user-images.githubusercontent.com/5703039/152975147-9d50d30a-e971-46de-933f-ec2e5021f881.png)

But anyways I specified dtype explicitly to `np.int64`. This should be enough for huge embedding tables (at least with sizes <= 2^63 ~= 9e+18)

Reply:
On Windows I have a different result: 
![image](https://user-images.githubusercontent.com/35459624/153055771-571f03e9-b1e6-474f-9143-512f1a27437d.png)

Please correct the code as I asked:)

Reply:
done